src/ttvchat/api/API.ts

```diff
@@ -0,0 +1,71 @@
+import * as request from 'request';
+
+import { TwitchKeys } from '../../enums';
+
+export class API {
+  public static async isUserFollowingChannel(userId: string, channel: string, token: string) {
+    const url = `https://api.twitch.tv/helix/users/followers?from_id=${userId}&to_name=${channel}`;
+    const result = await new Promise<boolean>((resolve, reject) => {
+      request.get(url, {
+        headers: {
+          'Authorization': `Bearer token`,
```

Comment:
Should this actually pass in the token value instead?

Reply:
Whoops... fixed up the string. It should have been `Bearer ${token}`

Reply:
Actually, the 'token' isn't required for this API so I removed it entirely.